api-documentation/invoice.md

```diff
@@ -820,6 +833,7 @@ Partner API Key.
   "direct_debit_iban": "...",
   "federation_membership_number": "...",
   "club_membership_number": "...",
+  "member_external_id": "...",
   "external_membership_number": "...",
   "customer": {
     "name": {
```

Comment:
:spiral_notepad: there is one thing I have noticed today:
All those 4 attributes above "customer" look like more fitting in to "customer", or? @vruizext